src/main/scala/chisel3/formal/FormalVerify.scala

```diff
@@ -0,0 +1,58 @@
+package chisel3.formal
+
+import java.io.PrintWriter
+import chisel3.RawModule
+
+object FormalVerify {
```

Comment:
I moved this logic out of FormalSpec to make it easier to write test cases for it.